src/ansys/geometry/core/designer/face.py

```diff
@@ -147,14 +152,25 @@ class Face:
         Active supporting Geometry service instance for design modeling.
     """
 
-    def __init__(self, id: str, surface_type: SurfaceType, body: "Body", grpc_client: GrpcClient):
+    def __init__(
+        self,
+        id: str,
+        surface_type: SurfaceType,
+        body: "Body",
+        grpc_client: GrpcClient,
+        is_reversed: bool = False,
+    ):
         """Initialize ``Face`` class."""
```

Comment:
```suggestion
        """Initialize the ``Face`` class."""
```
For all of these, you could redo as "Initialize an instance of the ``Face`` class" or "Initialize a ``Face`` instance.